tests/plugins/test_subcommands.py

```diff
@@ -46,10 +59,36 @@ def test_help(plugin, cli_main, capsys):
 
 
 def test_duplicated(plugin_manager, cli_main, capsys):
+    """
+    Ensures we get an error when attempting to register commands with the same `name` property
+    """
     plugin_manager.register(SubcommandPlugin())
     plugin_manager.register(SubcommandPlugin())
 
     cli_main()
     stdout, stderr = capsys.readouterr()
 
     assert "Conflicting `subcommands` plugins found" in stderr
+
+
+@pytest.mark.parametrize("builtin_command", BUILTIN_COMMANDS)
+def test_cannot_override_builtin_commands(builtin_command, plugin_manager, mocker):
+    """
+    Ensures that plugin subcommands do not override the builtin conda commands
+    """
+    # mocks
+    mocker.patch.object(SubcommandPlugin, "custom_command")
+    mock_log = mocker.patch("conda.cli.conda_argparse.log")
+
+    # setup
+    plugin = SubcommandPlugin(name=builtin_command)
+    expected_error = ArgumentParser._get_attempted_override_error(plugin.name, plugin.summary)
+    plugin_manager.register(plugin)
+
+    # run code under test
+    run(f"conda {builtin_command} --help")
+
+    # assertions; make sure we got the right error messages and didn't invoke the custom command
+    assert mock_log.error.mock_calls == [mock.call(expected_error)]
```

Comment:
Can we use `pytest.raises` instead?

Reply:
This should test the `log.error` call with [`caplog`](https://docs.pytest.org/en/7.1.x/how-to/logging.html#caplog-fixture)

Reply:
That wouldn't be possible because we do not raise an exception, right? 😅

Reply:
We just want to make sure the call to `logger.error` was done in the way we expected it to be (i.e. printing a message to the user about the attempt to override a built-in sub-command).

Reply:
oooh this is logger 🤦🏼‍♂️, can we use `caplog` instead?

https://docs.pytest.org/en/6.2.x/logging.html

Reply:
Sounds good. I was not aware of this fixture.

Reply:
@kenodegard , although this is a great idea, the implementation is proving to be exceedingly difficult. Therefore, I propose keeping the mocking behavior instead.